Add optional `DisplayTitle` prop for `TabContainer` tabs (#54)
sircfenner/StudioComponents: CHANGELOG.md

# Changelog

## Unreleased

> - Added an optional `DisplayTitle` prop to `TabContainer` children tabs to allow tabs to depend on some state for its display text

Reviewer: ```suggestion
- Added an optional `DisplayTitle` prop to `TabContainer` children tabs to allow displaying custom text on tabs
```
